Components/Navigation/navigation.css

```diff
@@ -0,0 +1,287 @@
+/*****  drawer *******/
+
+.drawer__menu {
+    width: 18vw;
+    overflow: scroll;
+}
+
+.drawer__title {
+    height: 8.5vh;
+    background: #000;
+    color: white;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    font-size: 1rem;
+    margin-bottom: 0.6rem;
+}
+
+.mail__add,
+.drawer__heading {
+    margin: auto;
+    cursor: pointer;
+}
+
+.textDrawer1 {
+    margin: 0 1rem;
+}
+
+.sidebar__item {
+    height: 5.4vh;
+    border: 1px solid white;
+    position: relative;
+    cursor: pointer;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.sidebar__item:hover {
+    background: rgb(200, 209, 236);
+}
+
+.sidebar__item a {
+    position: absolute;
+    color: rgb(13, 13, 112);
+}
+
+
+
+/******  main body ************/
+
+.body__heading {
+    height: 8.5vh;
+    background: purple;
+    color: white;
+    display: flex;
+    align-items: center;
+}
+
+.mainBody {
+    width: 100%;
+    overflow: scroll;
+}
+
+.drawerDescription {
+    font-size: 1rem;
+}
+
+h3 {
+    margin: 0 1.5rem;
+}
+
+.fa-bars {
+    margin: 0 2rem;
+    padding: 1rem;
+}
+
+.fa-bars:hover {
+    background: var(--primary-color);
+}
+
+
+
+/******** Permanent drawer ********/
+
+.permanent__body .body__heading {
+    position: sticky;
+    top: 0;
+}
+
+
+/********  Dismissible drawer ********/
+
+.dismissibleDrawer {
+    width: 0.0001rem;
+}
+
+.left {
+    display: flex;
+    align-items: center;
+}
+
+.dismissible__body .body__heading {
+    background-color: lightseagreen;
+}
+
+.dismissible__body .fa-bars:hover {
+    background: rgb(7, 151, 144);
+}
+
+/********  Modal drawer ***********/
+
+.drawer__Container--Modal {
+    position: relative;
+}
+
+.modalDrawer1 {
+    position: absolute;
+    height: inherit;
+    background: white;
+    visibility: hidden;
+    /* width: 18vw; */
+}
+
+.modal__body .body__heading {
+    background-color: coral;
+}
+
+.modal__body .fa-bars:hover {
+    background: rgb(236, 111, 65);
+}
+
+/**** icons  ****/
+
+span {
+    margin: 0 2rem 0 0;
+}
+
+span i {
+    cursor: pointer;
+}
+
+/**************************** end of nav *****************************************************/
+
+
+/* common */
+
+.center {
+    display: flex;
+    justify-content: center;
+}
+
+.wrap {
+    flex-wrap: wrap;
+}
+
+.h-4 {
+    font-size: 4rem;
+}
+
+.h-2 {
+    font-size: 2rem;
+}
+
+.h-1h {
+    font-size: 1.5rem;
+}
+
+.b-2 {
+    border: 2px solid gray;
+}
+
+.m-a {
+    margin: auto;
+}
+
+.sp-12 {
+    margin: 12vh 0 20vh 0;
+}
+
+.mt2-mb6 {
+    margin: 3vh 0 6vh 0;
+}
+
+.mv-4 {
+    margin: 4vh 0;
+}
+.mv-8 {
+    margin: 2vh 0 6vh 0;
+}
+.m-4 {
+    margin: 4vh;
+}
+
+.p-1 {
+    padding: 1rem;
+}
+
+.br-5 {
+    border-radius: 5px;
+}
+
+.fw-1 {
+    font-weight: bolder;
+}
+
+.gray {
+    color: gray;
+}
+
+/* mobile mode */
+
+@media only screen and (max-width: 800px) {
+    .drawer__menu {
+        width: 40vw;
+        border: 1px solid gray;
+    }
+
+    html {
```

Comment:
Font is too small on media query mobile view, I feel. Should atleast have font size similar to 'carbon code snippet' in that view.